store/models/orm.go

```diff
@@ -237,6 +237,16 @@ func (orm *ORM) BridgeTypeFor(name string) (BridgeType, error) {
 	return tt, err
 }
 
+// GetLastNonce retrieves the last known nonce in the database for an account
+func (orm *ORM) GetLastNonce(address common.Address) (uint64, error) {
```

Comment:
Let's get a simple table driven test for this with three scenarios: random error, storm.ErrNotFound, valid. Then we should be good to go 👍

Reply:
Random error might be too hard to produce. Skip if so.

Reply:
In the case of `storm.ErrNotFound` I think we should `return 0, nil`, because that's an expected case when you start a node for the first time.

Reply:
Ok @whatsadebugger, seems like this is the last action item left: table driven tests and returning `0, nil` on `storm.ErrNotFound`.